src/lib/components/Swap/Input.tsx

```diff
@@ -46,6 +46,31 @@ export interface InputProps {
   focused: boolean
 }
 
+interface UseInputOutputFormattedFieldArguments {
+  disabled: boolean
+  currencyAmount: CurrencyAmount<Currency> | undefined
```

Comment:
nit: I prefer to specify this (and below) as `currencyAmount?: CurrencyAmount<Currency>` instead of explicitly using `undefined`, as it's less verbose and usually the same.